tests/model_serving/model_server/conftest.py

```diff
@@ -154,3 +160,51 @@ def model_pvc(
 def skip_if_no_nfs_storage_class(admin_client: DynamicClient) -> None:
     if not StorageClass(client=admin_client, name=StorageClassName.NFS).exists:
         pytest.skip(f"StorageClass {StorageClassName.NFS} is missing from the cluster")
+
+
+@pytest.fixture(scope="class")
+def modelmesh_management_state(dsc_resource: DataScienceCluster) -> str:
+    return dsc_resource.instance.spec.components[DscComponents.MODELMESHSERVING].managementState
+
+
+@pytest.fixture(scope="class")
+def kserve_management_state(dsc_resource: DataScienceCluster) -> str:
+    return dsc_resource.instance.spec.components[DscComponents.KSERVE].managementState
+
+
+@pytest.fixture(scope="package")
+def skip_if_no_redhat_authorino_operator(admin_client: DynamicClient):
+    name = "authorino"
+    if not Authorino(
+        client=admin_client,
+        name=name,
+        namespace=f"{py_config['applications_namespace']}-auth-provider",
+    ).exists:
+        pytest.skip(f"{name} operator is missing from the cluster")
```

Comment:
This is not really correct, this means that the Authorino CR is missing from our namespace, but the operator itself could still be installed

Reply:
fixing the log in the next pr